Content.Server/Body/Components/InternalsComponent.cs

```diff
@@ -16,5 +16,13 @@ public sealed partial class InternalsComponent : Component
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("delay")]
         public float Delay = 3;
+
+        /// <summary>
+        /// Whether the mob should automatically attempt to activate its internals after spawning.
+        /// Intended for respirator mobs that may spawn in space or a spaced area.
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("auto")]
```

Comment:
```suggestion
        [DataField]
```
not needed

Reply:
Yeah fair I kinda forgot those are automatic now